python-lib/dku_io_utils/partitions_handling.py

```diff
@@ -99,20 +131,26 @@ def fix_date_elements_folder_path(partitions, types):
     return fixed_partitions
 
 
-def complete_file_path_time_pattern(dku_flow_variables, file_path_pattern):
+def complete_file_path_time_pattern(dku_flow_variables, file_path_pattern, source=None):
```

Comment:
Same comment as previously, I would advise using the terms input/output instead of origin/source/destination which are not used in DSS.

Reply:
Could we remove the need for dku_flow_variables as an argument, and instead retrieve it with the Dataiku API?